pom.xml

```diff
@@ -3,8 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-      <!--JENKINS-18537-->
-    <version>1.554.1</version>
+    <version>1.580.1</version>
```

Comment:
_[You need to be running a sufficiently recent Jenkins release: an LTS in the 1.580.x line or newer](https://github.com/jenkinsci/workflow-plugin#installation)_